apps/zui/src/domain/updates/mac-win-updater.ts

```diff
@@ -6,6 +6,7 @@ import {getMainObject} from "src/core/main"
 
 autoUpdater.autoDownload = false
 autoUpdater.autoInstallOnAppQuit = false
+autoUpdater.forceDevUpdateConfig = true
```

Comment:
I previously had not been using it since I always opt to test in personal fork repos, but during my test work I became aware of the `apps/zui/dev-app-update.yml` file. This allows for quick & dirty update testing in Dev mode since it allows us to fiddle with the current `version` string in `package.json` and the `owner` & `repo` settings in this YAML file and immediately see how the app would react if it had been bundled/released and was checking for an update at the configured GitHub Releases page. Well, per https://github.com/electron-userland/electron-builder/issues/6863 and the fix in https://github.com/electron-userland/electron-builder/pull/7117, within the jump period between our prior `electron-updater` version and the latest v6.2.1, they effectively broke it and now require setting this `forceDevUpdateConfig` in order for the file to be leveraged in Dev mode.